server/views/pages/explore.njk

```diff
@@ -44,7 +44,7 @@
 
   <div class="row row--small-top-padding">
     <div class="container">
-      <div class="grid">
+      <div class="grid grid--dividers">
```

Comment:
I think this class isn't set up to handle multiple rows of items very well – looks like the first divider is hidden by a `:first-child` selector, but the first on the second row has a divider to the left.